glutin/src/config.rs

```diff
@@ -164,9 +164,16 @@ impl ConfigTemplateBuilder {
         self
     }
 
-    /// The set of apis that are supported by configuration.
+    /// The set of apis that are supported by this configuration.
     ///
-    /// By default api isn't specified when requesting the configuration.
+    /// The default is specific based on the used API. If you
+    /// want to do config filtering based on the [`Api`] yourself, use [`Api::empty`].
+    ///
+    /// # Api-specific
+    ///
+    /// **EGL:** - [`Api::GLES2`] bit is set by default to avoid matching
+    /// [`Api::GLES1`] configs.
+    /// **GLX/WGL/CGL:** - [`Api::OPENGL`] is always present in the result.
```

Comment:
Does rustdoc render this nicely btw? I have a hunch the `-` has to be at the beginning to get an _Unordered List_ otherwise this just becomes one line/paragraph?

Reply:
You're right, there should be `-` to form a list.

Reply:
![image](https://github.com/rust-windowing/glutin/assets/2325264/8d5a0709-d53e-42e8-a064-98646757f83b)
Yup, it doesn't like it:

```suggestion
    /// - **EGL:** [`Api::GLES2`] bit is set by default to avoid matching
    /// [`Api::GLES1`] configs.
    /// - **GLX/WGL/CGL:** [`Api::OPENGL`] is always present in the result.
```